lib/turbolinks/redirection.rb

```diff
@@ -20,6 +20,26 @@ def redirect_to(url = {}, options = {})
       end
     end
 
+    def render(*args)
+      options = args.extract_options!
+      return super unless options[:turbolinks]
```

Comment:
if we turn this option into more of a `turbolinks if possible` then users could just sprinkle it into the `format.html` and would not need to worry about another response type ...

```
format.html { render turbolinks: true }
```

`return super unless options[:turbolinks] && request.xhr?`